### PR TITLE
Updating new vm name, add migrate, and Windows 10 limitation

### LINKED ISSUE #15454: Add migration material and Windows version support
- Add migration information back into the Windows installation
- Add note about lack of Windows 10 support

### PR DESCRIPTION
**Pull request by moxiegirl**
- Fixes #15454
- Add back in Migration material
- Rename `docker-vm` to `default`

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
@jeffdm @mchiang0610 Can I get some LGTMs here for the merge please

File: docs/installation/mac.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Docker container using standard localhost addressing such as `localhost:8000` or
 ![Linux Architecture Diagram](/installation/images/linux_docker_host.svg)
 
 In an OS X installation, the `docker` daemon is running inside a Linux VM called
-`docker-vm`. The `docker-vm` is a lightweight Linux VM made specifically to run
+`default`. The `default` is a lightweight Linux VM made specifically to run
```
Comment (mchiang0610) [inline docs/installation/mac.md]: `default`. The `default` VM is a lightweight Linux VM made specifically to run

**Comment by mchiang0610** [comment]
@moxiegirl LGTM!  Thanks

**Comment by moxiegirl** [comment]
author LGTM

**Comment by moxiegirl** [comment]
@calavera Cherry pick this one please for install docs

**Comment by calavera** [comment]
cherry picked into #15091

**Pull request merged**